examples/platform/silabs/OTAConfig.cpp

```diff
@@ -17,10 +17,18 @@
  */
 
 #include "OTAConfig.h"
+#include <app/server/Server.h>
+
+#ifndef SIWX_917
 
 #include "application_properties.h"
-#include <app/server/Server.h>
 
+#if defined(SL_COMPONENT_CATALOG_PRESENT)
+#include "sl_component_catalog.h"
+#endif
+
+// Only include app properties if the Gecko SDK component that does it automatically isn't present
+#if !defined(SL_CATALOG_GECKO_BOOTLOADER_INTERFACE_PRESENT)
```

Comment:
This change is correct. 
Note that we need this now because SL_CATALOG_GECKO_BOOTLOADER_INTERFACE_PRESENT was removed from the sdk_support repo in this PR: https://github.com/SiliconLabs/sdk_support/pull/119